### DISCUSSION TITLE
Trouble with `description` environment in custom latex

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#4330] [Q&A] [answered]
For my book, I need to have a second title page with ISBN, CC BY License, etc. according to some editorial standards. 

The best solution I could find was to use the `include-before-body` in pdf and html formats. It's a bit redundant, because I have to present the same info, but in latex/html (the included info). I think it would be good for Quarto if in the future, books could have this structure, as a second page is pretty common.

Now, I have a question because when I defined this in LaTeX, I am having trouble with indentation in the `\begin{description}` environment. Here's a screen grab of the PDF output:

![image](https://user-images.githubusercontent.com/7606540/218346067-920fd70a-46f5-47fd-aa54-ffd06047c8e4.png)

When I run my code in Overleaf, the two red lines are the same. That is, the first lines of the `\item[]`s are not indented, which is how the editorial standards request I do it. I've tried tweaking the `labelindent=0cm`, but Quarto spews errors about `compilation failed- missing packages (automatic installed disabled)  
LaTeX Error: Something's wrong--perhaps a missing \item.`

Here's a MWE for the second page used by `include-body-before:` 

```latex
\noindent Words are here.

\begin{description}[labelindent=0cm,leftmargin=1cm]
  \item [Ce manuel]est utilisé dans le cadre du cours \emph{Analyse et
conception de logiciels (LOG210)} à l'École de technologie supérieure
(ÉTS).
\end{description}
```

I'm guessing somewhere Quarto has redefined the `description` environment and those options aren't legal anymore, but I couldn't find where. If anyone has some hints, I would be very grateful.

**Comment by mcanouil** [comment, marked as answer]
You are talking about two different things which is not the best, _i.e_, a feature request and a question about a problem.
I recommend you separate the feature request from your issue if you want it to be seen as is.

Could you share a minimal (not) working example? Currently, others and/or I have to try to write an example to reproduce while you already have one ...

**Reply by fuhrmanator** [reply]
Alright, while trying to make a MWE for the unusual indentation, I found the problem.

```yalm
  pdf:

    pdf-engine: pdflatex
    classoption: 
      # pass french to book class (babel will see it automatically)
      - french
```

Turning off the `- french` option fixes the problem (although I need that for my book project). This bug is out of the scope of Quarto, so we can call this discussion resolved.

I will try to find the other discussion about book structure and add the comment about the second page there.

**Reply by fuhrmanator** [reply]
Here are more details. I scanned the documentation for [French babel (in French)](https://mirror.its.dal.ca/ctan/macros/latex/contrib/babel-contrib/french/frenchb-doc.pdf), and there is a lot of discussion about how indentation of lists is modified.

I added to the `template.tex` (which I load in my _quarto.yml) the following line, just after [the loading of `babel`](https://github.com/quarto-dev/quarto-cli/blob/7b53645bd0a525674e034264ff5a17607110966d/src/resources/formats/pdf/pandoc/template.tex#L228-L236):

```tex
\setlength{\descindentFB}{0cm}
```

It solves the problem in Quarto, at least for the case I mentioned in my first question.
